modules/gapi/include/opencv2/gapi/gproto.hpp

```diff
@@ -57,6 +57,7 @@ template<class Tag>
 struct GIOProtoArgs
 {
 public:
+    GIOProtoArgs() = default;
```

Comment:
```
PyObject* pyobj_ins = NULL;
GProtoInputArgs ins;
PyObject* pyobj_outs = NULL;
GProtoOutputArgs outs;

const char* keywords[] = { "ins", "outs", NULL };
if( PyArg_ParseTupleAndKeywords(py_args, kw, "OO:GComputation", (char**)keywords, &pyobj_ins, &pyobj_outs) &&
    pyopencv_to(pyobj_ins, ins, ArgInfo("ins", 0)) &&
    pyopencv_to(pyobj_outs, outs, ArgInfo("outs", 0)) )
 {
    new (&(self->v)) Ptr<cv::GComputation>(); // init Ptr with placement new
    if(self) ERRWRAP2(self->v.reset(new cv::GComputation(std::move(ins), std::move(outs))));
    return 0;
 }
```

Here is a code generated for `GComputation ctor`, so it requires default ctor for `GIOProtoArgs`

Reply:
Got it